app/components-react/windows/NotificationsAndNews.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from 'react';
+import { Menu } from 'antd';
+import { useVuex } from 'components-react/hooks';
+import { ModalLayout } from 'components-react/shared/ModalLayout';
+import { Services } from 'components-react/service-provider';
+import News from './News';
+import Notifications from './Notifications';
+import { $t } from 'services/i18n';
+import styles from './NotificationsAndNews.m.less';
+
+export default function NotificationsAndNews() {
+  const { NotificationsService } = Services;
+
+  const tabNames = {
+    news: $t('News'),
+    notifications: $t('Notifications'),
+  };
+
+  const [selectedTab, setSelectedTab] = useState(tabNames.news);
+
+  const { notificationGroups } = useVuex(() => ({
+    notificationGroups: {
+      unread: NotificationsService.views.getUnread(),
+    },
+  }));
+
+  useEffect(() => {
+    if (!!notificationGroups && notificationGroups.unread.length > 0) {
+      setSelectedTab(tabNames.notifications);
+    }
+  }, [notificationGroups]);
+
+  return (
+    <ModalLayout hideFooter>
+      <div className={styles.notificationsMenu}>
+        <Menu
```

Comment:
I think we should switch to the `<Tabs>` component.  It's much simpler than the `<Menu>` component for implementing tabs and will reduce the complexity here a bit.  It keeps track of the currently active tab and has a simple prop for the default tab.

Reply:
Done! And yes, definitely more elegant.